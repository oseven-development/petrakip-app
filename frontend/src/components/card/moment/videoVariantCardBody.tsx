export interface VideoVariantProps {
  variant: 'video'
  videoSrc: string
}

export const VideoVariantCardBody: React.FC<VideoVariantProps> = ({
  videoSrc,
}) => (
  <video width="100%" controls>
    <source src={videoSrc} type="video/mp4" />
    Your browser does not support the video tag.
  </video>
)
