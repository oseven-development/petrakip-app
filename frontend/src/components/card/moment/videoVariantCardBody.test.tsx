import { render } from '@testing-library/react'

import { VideoVariantCardBody } from './videoVariantCardBody'

describe('VideoVariantCard Component', () => {
  const audioVariantCard = (
    <VideoVariantCardBody variant="video" videoSrc="mySource" />
  )

  test('renders without crashing', () => {
    const { container } = render(audioVariantCard)
    expect(container).toMatchSnapshot()
  })
})
