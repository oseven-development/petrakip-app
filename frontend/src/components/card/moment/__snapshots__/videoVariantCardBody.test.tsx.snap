// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoVariantCard Component renders without crashing 1`] = `
<div>
  <video
    controls=""
    width="100%"
  >
    <source
      src="mySource"
      type="video/mp4"
    />
    Your browser does not support the video tag.
  </video>
</div>
`;
